client/src/header/components/Header.tsx

```diff
@@ -54,6 +55,7 @@ const Header: FC<Props> = ({ onLogin, isLoggedIn, name, onLogout }) => {
   );
 
   return (
+    receivedError ? <Redirect to='/error'/> :
```

Comment:
So as it stands, I think this will always call an error for now. I can comment out this line of code if you guys don't want this to appear every time, but otherwise, we'll just see the message every time until the serve code is actually run with the client code.